.chronus/changes/improve completion-support-in-ide-2024-4-6-16-27-17.md

```diff
@@ -0,0 +1,10 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/compiler"
+---
+
+Support completion for Model Expression, Object/Array Literal as decorator argument
+Support completion for Object/Array Literal in const assignment with type definition
+Support completion for Object/Array Literal as scalar constructor argument
+Support completion for Model with extended properties
```

Comment:
````suggestion
Support completion for object values and model expression properties.

  Example
  ```tsp
  model User {
    name: string;
    age: int32;
    address: string;
  }

  const user: User = #{name: "Bob", ┆}
                                    | [age]
                                    | [address]
  ```

````

Reply:
And I would create a copy of this file to describe `Support completion for Model with extended properties`

```md
Add completion of base model properties 
```

maybe with an example as above